cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -41,6 +41,8 @@
     $crontab.join("\n")
   end
 
+  # CRONJOB TIME SYNTAX: https://help.ubuntu.com/community/CronHowto
```

Comment:
Feel welcome to suggest another URL, this was chosen as it seemed to best follow our syntax (we don't support days of week as strings, only numbers), give useful examples, and seem like the page may last.